### DISCUSSION TITLE
Why Godot can't be Linked to Payoneer or Paypal

### DISCUSSION THREAD

**Discussion by IbrahimSemab** [godotengine/godot-proposals#5926] [General Discussions]
I was wondering if Godot supported linking digital payment services providers like Paypal, Payoneer etc. It is a really good feature that must be added through may be plugin that will allow developers to add billing through Payoneer or any other without having Google Play Console account.

**Comment by Calinou** [comment]
Supporting payment services is something that should be done with third-party add-ons, not in the core engine.

**Reply by IbrahimSemab** [reply]
Yeah, but there is no plugin or third-party add-ons supporting payment services.

**Reply by Zireael07** [reply]
Googling for "Godot admob addon" shows at least two

**Comment by Mikeysax** [comment]
You could probably utilize a webview and then integrate with any payment processor with JavaScript

ie
https://github.com/vini-guerrero/Godot-Webview

You could also just connect to any payment processor and utilize their JavaScript APIs directly:

https://docs.godotengine.org/en/stable/classes/class_javascript.html

Or call via system calls into python / curl lib or some other language if the payment processor supports it or just make HTTP requests directly:

https://docs.godotengine.org/en/stable/tutorials/networking/http_request_class.html?highlight=http
